scripts/docker/19.03.sh

```diff
@@ -0,0 +1,531 @@
+#!/bin/sh
+set -e
```

Comment:
Maybe add a comment for where this file came from - in case we want to update it?